lib/open_mic.rb

```diff
@@ -0,0 +1,17 @@
+class OpenMic
+  attr_reader :location, :date, :performers
+  def initialize(location:, date:)
```

Comment:
I'd actually have to take a look at this on a machine to know what this does--you've basically set up two symbols as parameters, instead of variables. I think I expect this to fail all of your tests, but maybe this is some "Ruby loves you" magic at work here, coming to your rescue.  🙂 

In any case, the interaction pattern is telling us that open mics are created with a single argument `open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})`, which is a hash of information.  so you'd have to do this similarly to how you saw me update the Unicorn class we did at the end of the hash lesson this week.  You'll see this in the walkthrough this morning, but let me know if you still have questions afterwards.